pkg/api/write.go

```diff
@@ -24,6 +24,85 @@ import (
 	"github.com/timescale/promscale/pkg/util"
 )
 
+type writeStage func(http.ResponseWriter, *http.Request, *Metrics) bool
```

Comment:
why was all of this moved? Are there any changes in the moved code?

Reply:
it's just moved, without any changes. The reason being easier navigation within `api/write.go`, as we can click on can stage function to go directly to the function definition.